feature/qos/ate_tests/bursty_traffic_test/bursty_traffic_test.go

```diff
@@ -704,9 +704,10 @@ func ConfigureQoS(t *testing.T, dut *ondatra.DUTDevice) {
 	for _, tc := range classifierIntfs {
 		i := q.GetOrCreateInterface(tc.intf)
 		i.SetInterfaceId(tc.intf)
-		if deviations.ExplicitInterfaceRefDefinition(dut) {
-			i.GetOrCreateInterfaceRef().Interface = ygot.String(tc.intf)
-			i.GetOrCreateInterfaceRef().Subinterface = ygot.Uint32(0)
+		i.GetOrCreateInterfaceRef().Interface = ygot.String(tc.intf)
+		i.GetOrCreateInterfaceRef().Subinterface = ygot.Uint32(0)
+		if deviations.InterfaceRefConfigUnsupported(dut) || deviations.IntfRefConfigUnsupported(dut) {
```

Comment:
to my understanding  deviations are for interfaceref/sub-interface and interfaceref/interface. Right? Can we rename the deviations to reflect what they are for? InterfaceRef and IntfRef both have the same meaning. Also, we need to handle each deviation separately since it is possible the device requires interfaceref/interface to be set, but does not support interfaceref/sub-interface or sub-interface 0.

Reply:
it is consistent with the current deviation used.
reference: https://github.com/openconfig/featureprofiles/pull/1768
changing name of deviations should not be under the scope of our pull.

Reply:
you are changing all the test in one pull, so that would be great if you also improve the misleading names and handle the deviations properly.
Also, concerning the the scope and the guideline, we open one pull per test,  can u please follow the guideline. Also, please do not resolve the comment if you have not resolved.  

@vosipchu FYI